tests/integration/stepfunctions/utils.py

```diff
@@ -0,0 +1,10 @@
+import os
+from typing import Final
+
+_THIS_FOLDER: Final[str] = os.path.dirname(os.path.realpath(__file__))
+
+
+def lambda_handler(file_name: str) -> str:
+    if not file_name.endswith(".py"):
+        file_name += ".py"
+    return os.path.join(_THIS_FOLDER, f"lambda_functions/{file_name}")
```

Comment:
I know it seems a bit unnecessary at first but I'd suggest staying close to the pattern we're using in lambda and deleting the content of this file.

This is purely out of a practical sense that PyCharm otherwise can't resolve the path and won't jump to the corresponding file.


Instead the tests would manually build this path each time it needs a file e.g. `os.path.join(os.path.dirname(__file__), "./lambda_functions/id_function.py")`

Refactoring this in the CloudFormation tests was also a massive pain and works much better just referencing directly from the test that's using it.